packages/core/src/styles/themes/themes.stories.ts

```diff
@@ -364,15 +330,23 @@ export function dynamicTheme() {
           </cds-input>
 
           <cds-range layout="vertical">
-            <label>Adjust scale</label>
-            <input type="range" value="100" @input=${updateScale} min="90" max="120" />
-            <cds-control-message class="scale-label">Scale: 100%</cds-control-message>
+            <label>Space Scale (component size)</label>
+            <input type="range" value="100" @input=${updateScale} min="70" max="140" />
+            <cds-control-message class="scale-label">--cds-global-scale-space: 1 (100%)</cds-control-message>
+          </cds-range>
+
+          <cds-range layout="vertical">
+            <label>Layout Space Scale (whitespace density)</label>
+            <input type="range" value="100" @input=${updateDensity} min="70" max="140" />
+            <cds-control-message class="density-label">--cds-global-scale-layout-space: 1 (100%)</cds-control-message>
           </cds-range>
 
           <cds-range layout="vertical">
-            <label>Adjust Density</label>
-            <input type="range" value="100" @input=${updateDensity} min="70" max="150" />
-            <cds-control-message class="density-label">Density: 100%</cds-control-message>
+            <label>Typography Scale (font size)</label>
+            <input type="range" value="100" @input=${updateScaleTypography} min="90" max="120" />
+            <cds-control-message class="scale-typography-label"
+              >--cds-global-scale-typography: 1 (100%)</cds-control-message
+            >
```

Comment:
With the scale multiplier tokens we can control the three main types of space in the system (whitespace, component size and typography) rather easily.